### DISCUSSION TITLE
Collaboration app for npm

### DISCUSSION THREAD

**Discussion by mihhu** [excalidraw/excalidraw#5902] [General]
Hello,

This is Mihaela from the Jitsi team.
Thank you for your relentless effort to develop and improve this whiteboard solution! At Jitsi we found it a great fit with our vision and we integrated it in our meetings. The implementation required some tweaks, such as converting your collab app to a React component and hiding some features that would seem overwhelming in a video call solution. We’d also like to make firebase an opt-in thing.

Here’s our take: https://github.com/jitsi/excalidraw/tree/jitsi.
And here you can see it in action: https://meet.jit.si/ join a room and enable it from the toolbar > More menu > Open whiteboard.

We’re interested in contributing to bringing these changes incrementally to your repo. Would this be of interest to you?

**Comment by saghul** [comment]
Bumping this in case it was missed. We are still keen on it.

**Comment by maymage** [comment]
@ad1992 this could really improve the experience over at jitsi. Would you comment or close this issue, if there's no interest on excalidraw's side?
